### PR TITLE
Added review model and migration.

### PR DESCRIPTION
**Pull request by Jgorin**
Changes:

- Added review model
- added review migration
- added review validations

### CODE REVIEW

**Review by kaydc18 — CHANGES_REQUESTED** [review]
Some things... I noticed you didn't create an association for your animals table. Also as a PR, this is a little too small. How is the animals model affected, and where is this code being used as part of a whole feature?

**Comment by Jgorin** [comment]
changes:

- added review model/migration
- added list of reviews to animal show page

**Comment by Jgorin** [comment]
changes:

-tweaked rspec tests for animal show page to include reviews

File: app/models/animal.rb [hunk 1]
```diff
@@ -2,5 +2,6 @@ class Animal < ApplicationRecord
   validates :name, presence: true
   validates :body, presence: true
   validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }
+  has_many :reviews
```
Comment (kaydc18) [inline app/models/animal.rb:5]: Add spacing between validations and associates like your review model

File: spec/features/controllers/api/v1/animals_controller_spec.rb [hunk 2]
```diff
@@ -14,13 +14,25 @@
     photo_path: "./spec/support/images/photo.png"
   ) }
 
+  let!(:user1) {User.create(
+    username: "josh",
+    password: "111111",
+    email: "jgorin@conncoll.edu"
+  )}
```
Comment (kaydc18) [inline spec/features/controllers/api/v1/animals_controller_spec.rb:21]: A user is created, but not used? You would need this in the spec if a user needed to be logged in to see the page/create a review etc.

**Review by kaydc18 — CHANGES_REQUESTED** [review]
Sorry all, a few things!

File: spec/features/controllers/api/v1/animals_controller_spec.rb (hunk 2 above)
Comment (Jgorin) [reply spec/features/controllers/api/v1/animals_controller_spec.rb:21]: We needed to create a user object in order to create a new review

```
  let!(:review) {Review.create(
    title: "bad",
    description: "soo bad",
    rating: 1,
    animal: first_animal,
    user: user1
  )}
```

**Comment by Jgorin** [comment]
Changes:

-changed animal state to contain reviews

**Review by kaydc18 — APPROVED** [review]
NOICE!

**Pull request merged**